File: data/src/main/java/org/oppia/data/backends/gae/model/SubtitledHtml.java
package org.oppia.data.backends.gae.model;

import com.squareup.moshi.Json;

public class SubtitledHtml {

  /*
   * Parent Hierarchy: ExplorationContainer -> Exploration -> State -> InteractionInstance -> Hint
   */

  //TODO: HTML-String data type
  @Json(name = "html")
  public String html;

  @Json(name = "content_id")
  public String contentId;

}

File: data/src/main/java/org/oppia/data/backends/gae/model/SubtitledHtml.kt
package org.oppia.data.backends.gae.model

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class SubtitledHtml(

  @Json(name = "html") val html: String,
  @Json(name = "content_id") val contentId: String

)
